### ISSUE TITLE
Unable to bind NgModel to Select with static options

### ISSUE THREAD

**Issue by detchison** [sherweb/ng2-materialize#175]
We have tried multiple ways of doing this, but it doesnt seem to be working. Whenever we try to bind a select with ngModel the (already populated) initial value never gets set. On the example project it does... I cant tell what is different, I know there is a thrid part issue with the modal options (materializecss causing that) but with this I am at a loss... this is an absolute must have, and its not working as intended, is there some sort of configuration i am missing?

```html
<mz-select-container class="col s6">
  <select mz-select
    id="options-select"
    [label]="'Label'"
    [placeholder]="'Select a Title'"
    [(ngModel)]="editTherapist.title">
    <option [ngValue]=1>BCBA</option>
    <option [ngValue]=2>RBT</option>
  </select>
</mz-select-container>
```

**Comment by scote** [comment]
You should use `[value]`. `[ngValue]` is used for object type.

**Comment by detchison** [comment]
Doesn't work either way, we have moved onto using Reactive forms and setting the values that way

**Comment by detchison** [comment]
This solution is not working either, I made a blank angular-cli project, following all the instructions, and made 2 selects, one plain html select, and an mz-select, I was able to bind the selected value to the regular html select with ngModel or ReactiveForms, but was not able to bind the value on the mz-select using either method...

**Comment by jfcere** [comment]
Hi @detchison,

Thanks for reporting the issue. With your code sample I've been able to reproduce the problem. The demo application example that works with `ngModel` uses `ngFor` on options. If I use it on your example it works great. We are going to fix it with static values but in the meanwhile you could use it that way.

##### example.component.html
```html
<mz-select-container class="col s6">
  <select mz-select
    id="options-select"
    [label]="'Label'"
    [placeholder]="'Select a Title'"
    [(ngModel)]="editTherapist.title">
    <option *ngFor="let title of titles" [value]="title.value">{{ title.text }}</option>
  </select>
</mz-select-container>
```

##### example.component.ts
```typescript
titles = [
  { value: 1, text: 'BCBA' },
  { value: 2, text: 'RBT' },
];

editTherapist = {
  title: 1,
};
```

**Comment by detchison** [comment]
Thanks a lot, I appreciate it, love it so far, just some hiccups here and there and while I don't know where the problem lies... glad to point them out.

**Comment by bakoserge** [comment]
Is this issue fixed? I was using the *ngFor which was working fine and for some reason stopped working. 

`<mz-select-container>
                                                <select [label]="Do you have a pregnancy at risk" mz-select id="pregnancy-select" [(ngModel)]="riskyPregnancy" [ngModelOptions]="{standalone: true}">
                                                    <option *ngFor="let option of confirmOptions" [value]="option.value">{{option.label}}</option>
                                                </select>
                                            </mz-select-container>`

`public confirmOptions: any = [
        { value: true, label: 'Yes' },
        { value: false, label: 'No' }
      ]`

**Comment by CarlosAyala** [comment]
Hi!

I have a similar issue, my code is:

```
<mz-select-container>
            <select mz-select [(ngModel)]="address_selected"
              id="shipping-address"
            >
                <option value=""> New Address </option>
                <option *ngFor="let address of address_user" [value]="address.id.toString()">{{address.first_name}},{{address.last_name}},{{address.address_line}}</option>
            </select>
        </mz-select-container>
```

If I change the value of "address_selected" to empty string "", the select combo don't select the "New Address" option.

**Comment by jfcere** [comment]
Hi @CarlosAyala,

As a workaround, I suggest you to add the "New Address" option to your list of "address_user".

Your biggest challenge will be for the option text as you use the interpolation `{{address.first_name}},{{address.last_name}},{{address.address_line}}` to concate multiple properties.

The option text could be a property of `address_user` computed in your code to facilitate the fact that "New Address" does not have the `first_name\last_name\address_line` properties.

**Comment by CarlosAyala** [comment]
Hi @jfcere 
I fix this problem changing the "" value to "-1" and with ng2-materialize version 1.5.2.

**Comment by jfcere** [comment]
NgModel binding with static options has been fixed and will be released in [v1.6.0](https://github.com/sherweb/ng2-materialize/milestone/2) which should come early october.
